Orange/widgets/unsupervised/owdistancefile.py

```diff
@@ -81,7 +84,7 @@ def browse_file(self, in_demos=0):
             start_file = self.last_path() or os.path.expanduser("~/")
 
         filename = QtGui.QFileDialog.getOpenFileName(
-            self, 'Open Distance File', start_file)
+            self, 'Open Distance File', start_file, "(*.dst *.txt)")
```

Comment:
I thought I had some .txt files with distances, so I assumed this used to be a valid extension and wanted to support it. Stupid. I removed it.